examples/redirects/pages/news/[...slug].js

```diff
@@ -0,0 +1,33 @@
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+import styles from '../../styles.module.css'
+
+const Code = (p) => <code className={styles.inlineCode} {...p} />
+
+const News = ({ props }) => {
+  const { asPath, route, query } = useRouter()
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.card}>
+        <h1>Path: {asPath}</h1>
+        <hr className={styles.hr} />
+        <p>
+          You were redirected here from <Code>/old-blog/first-post</Code>
```

Comment:
@lfades Could you give me pointers on how to dynamically show the `redirected from` path here ? From what i know i could only think of using something along the way of context api, I just wanted to know if there are other better ways.

Reply:
@robintom You can't and there's no need to do that 👍